ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+library = foreman-ansible-modules/modules
+module_utils = foreman-ansible-modules/module_utils
+roles_path = $PWD/galaxy_roles:$PWD/roles
+inventory = inventory
```

Comment:
I'd recommend making an inventories/ directory and putting the sample in there.

Reply:
I like it too, even @flatfender was suggesting the same, I just followed this from Ansible and so kept it at top-level: 
https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#directory-layout

Reply:
ok, I will make this as `inventory_files/inventory`, also would move `inventory.sample` file to `inventory_files`, else suggest me a name :)